### ISSUE TITLE
Is the phone cursed?

### ISSUE THREAD

**Issue by nafck** [College-Kings/College-Kings#242]
Again, I started CK1 from the beginning, and I decided to use the developer console to jump to certain scenes (this is because I initially wanted to report a possible plot hole on Penelope's path). So, I jumped to her trial (from somewhere in v1, at the very beginning, during the Julia driving part), then after I defended her, I jumped again to the airport part (v11s23 with Amber and Penelope) and after I clicked on the phone, this error showed up:

```
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/v11/scene12.rpy", line 197, in script call
    u "Hmm... got some new clothes, had a hearing with the dean, and well, Emily said she no longer wants to talk to me."
  File "game/phone/phone.rpy", line 44, in script
    call screen phone
  File "renpy/common/000statements.rpy", line 609, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
Exception: VPGrid not completely full, needs a multiple of 3 children.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/v11/scene12.rpy", line 197, in script call
    u "Hmm... got some new clothes, had a hearing with the dean, and well, Emily said she no longer wants to talk to me."
  File "game/phone/phone.rpy", line 44, in script
    call screen phone
  File "D:\renpy-8.0.1-sdk\renpy\ast.py", line 2232, in execute
    self.call("execute")
  File "D:\renpy-8.0.1-sdk\renpy\ast.py", line 2220, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "D:\renpy-8.0.1-sdk\renpy\statements.py", line 281, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 609, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "D:\renpy-8.0.1-sdk\renpy\exports.py", line 3161, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
  File "D:\renpy-8.0.1-sdk\renpy\ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "D:\renpy-8.0.1-sdk\renpy\display\core.py", line 3377, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, **kwargs) # type: ignore
  File "D:\renpy-8.0.1-sdk\renpy\display\core.py", line 3804, in interact_core
    root_widget.visit_all(lambda i : i.per_interact())
  File "D:\renpy-8.0.1-sdk\renpy\display\core.py", line 582, in visit_all
    d.visit_all(callback, seen)
  File "D:\renpy-8.0.1-sdk\renpy\display\core.py", line 582, in visit_all
    d.visit_all(callback, seen)
  File "D:\renpy-8.0.1-sdk\renpy\display\core.py", line 582, in visit_all
    d.visit_all(callback, seen)
  File "D:\renpy-8.0.1-sdk\renpy\display\screen.py", line 456, in visit_all
    self.child.visit_all(callback, seen=None)
  File "D:\renpy-8.0.1-sdk\renpy\display\core.py", line 582, in visit_all
    d.visit_all(callback, seen)
  File "D:\renpy-8.0.1-sdk\renpy\display\core.py", line 582, in visit_all
    d.visit_all(callback, seen)
  File "D:\renpy-8.0.1-sdk\renpy\display\core.py", line 584, in visit_all
    callback(self)
  File "D:\renpy-8.0.1-sdk\renpy\display\core.py", line 3804, in <lambda>
    root_widget.visit_all(lambda i : i.per_interact())
  File "D:\renpy-8.0.1-sdk\renpy\display\viewport.py", line 678, in per_interact
    raise exc
Exception: VPGrid not completely full, needs a multiple of 3 children.

Windows-10-10.0.22000 AMD64
Ren'Py 8.0.1.22070801
College Kings 1.1.2 (SHA: b'3d4d75f20')
Fri Aug  5 15:20:20 2022
```

I had to press reload to display the phone normally, but after that, it poped up normally every time I clicked on it

**Comment by nafck** [comment]
UPDATE: It doesn't happen only on that scene apparently

**Comment by KiloOscarSix** [comment]
This should be fixed in v1.2.0 with the phone being submodules from ck2.

If this error persists please reopen this issue :)
